Translations.cs

```diff
@@ -1278,7 +1278,7 @@ private void Awake()
             itemDescrTranslation[3113] = "Used to train\n(%6%)Stamina\n(%6%)Strength";
             itemNamesTranslation[3113] = "Stone of Procyon";
 
-            itemDescrTranslation[3114] = "Used to train\n(%6%)Wisdom\n(%6%)Intelligence";
+            itemDescrTranslation[3114] = "Used to train\n(%6%)Stamina\n(%6%)Intelligence";
```

Comment:
I see you changed training stones values in Playerstats (case 3114:) but description stats didnt change:
Used to train\n(%6%)Stamina\n(%6%)Intelligence = 6% Stamina / 6% Intelligence

Reply:
Done